app/renderer/polyfills/index.js

```diff
@@ -0,0 +1,37 @@
+let log, settings, clipboard, shell, remote, ipcRenderer,
+    i18NextBackend, i18NextBackendOptions;
+
+function buildForBrowser () {
+  if (process.env.BUILD_BROWSER) {
+    return true;
+  }
+
+  if (typeof navigator !== 'undefined') {
```

Comment:
this could be simplified to `return typeof navigator !== 'undefined' && !/electron/i.test(navigator.userAgent)`

Reply:
ah yes good point. i was debugging something and left the debugging structure.